### DISCUSSION TITLE
tags not found

### DISCUSSION THREAD

**Discussion by jiajunngjj** [mmistakes/minimal-mistakes#2656] [Q&A] [answered]
Hi, 

I'm getting 404 when routing to my tags. I scanned through the repo, looks fine to me. Any idea?

Error screenshot:
![image](https://user-images.githubusercontent.com/25560159/90241262-450c9a00-de5d-11ea-8f95-5960ff946ebc.png)

[blog](https://jiajunngjj.github.io/blog)
[repo](https://github.com/jiajunngjj/blog)

**Comment by iBug** [comment, marked as answer]
Read [the documentation](https://mmistakes.github.io/minimal-mistakes/docs/configuration/#archive-settings) carefully:

> **Note**: these are simply hash (fragment) links into the full taxonomy index pages. For them to resolve properly, the category and tag index pages need to exist at `/categories/index.html` (copy to `_pages/category-archive.md`) and `/tags/index.html` (copy to `_pages/tag-archive.md`).

**Comment by jiajunngjj** [comment]
@iBug Ah, I must have missed this out. Ok now it works like a charm. Thanks for the pointer!
